### PR TITLE
Vertx upgrade to v4.4.4

### PR DESCRIPTION
**Pull request by lceni**

### CODE REVIEW

**Review by qGYdXbY2 — APPROVED** [review]

File: xyz-hub-service/src/main/resources/openapi-recipes.yaml [hunk 1]
```diff
@@ -120,7 +120,7 @@ recipes:
     replace:
       - type: value
         path: servers.0.url
-        replace: /
+        replace: http://localhost:8080/
```
Comment (roegi) [inline xyz-hub-service/src/main/resources/openapi-recipes.yaml:123]: When being deployed on a different server, will this work as well?

File: xyz-hub-service/src/main/resources/openapi.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ info:
   contact: {}
   version: 1.0.0
 servers:
-  - url: /hub
+  - url: http://localhost:8080/hub
```
Comment (roegi) [inline xyz-hub-service/src/main/resources/openapi.yaml:9]: When being deployed on a different server, will this work as well?

**Review by roegi — COMMENTED** [review]
Added some comment inline.

File: xyz-hub-service/src/main/resources/openapi-recipes.yaml (hunk 1 above)
Comment (lceni) [reply xyz-hub-service/src/main/resources/openapi-recipes.yaml:119]: yes, the new router included an URL-format validation, however it does not influence in the routing.

File: xyz-hub-service/src/main/resources/openapi.yaml (hunk 2 above)
Comment (lceni) [reply xyz-hub-service/src/main/resources/openapi.yaml:9]: yes

**Review by roegi — APPROVED** [review]

**Pull request merged**